daphne/src/vdaf/mod_test.rs

```diff
@@ -138,9 +140,18 @@ async fn roundtrip_report() {
     }
 }
 
-#[test]
-fn roundtrip_report_unsupported_hpke_suite() {
-    let t = Test::new(TEST_VDAF);
+#[tokio::test]
+async fn roundtrip_report_draft02() {
```

Comment:
Ok, now I see what you were getting at! Go ahead and de-dup the mod_test.rs tests as you suggested.